### PR TITLE
MULTIARCH-4696: modify script to get successful run

### PR DESCRIPTION
**Pull request by pkenchap**

### CODE REVIEW

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-powervs-ovn-multi-ppc64le

File: ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh [hunk 1]
```diff
@@ -327,7 +328,7 @@ function build_upi_cluster() {
                     then
                     OUTPUT="yes"
                     fi
-                done || true
```
Comment (prb112) [inline ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh:330]: .

**Pull request merged**